includes/definitions/horizon.yaml

```diff
@@ -0,0 +1,10 @@
+os: horizon
```

Comment:
Is this definitely the best os name? Is the firmware name?

Reply:
Horizon would be the best of name IMO.